### PR TITLE
Add UI frontend article

### PR DESCRIPTION
**Pull request by bucanero**

### CODE REVIEW

**Review by calebjacob — APPROVED** [review]

**Review by mpeterdev — CHANGES_REQUESTED** [review]
please hold on merge, I am going to suggest some additions/modifications

File: docs/2.build/3.near-components/what-is.md [hunk 1]
```diff
@@ -8,6 +8,14 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import {WidgetEditor} from "@site/src/components/widget-editor"
 
+:::warning What is the state of BOS (NEAR Components)?
+
+We no longer recommend building on BOS due to its limited capabilities and discontinued security maintenance. Developers with active projects on BOS are encouraged to migrate to another deployment strategy.
+
+See [here](/build/web3-apps/frontend#bos-socialvm) for more info
```
Comment (bucanero) [inline docs/2.build/3.near-components/what-is.md:15]:
```suggestion
See [here](/build/web3-apps/frontend#bos-socialvm) for more information.
```

File: docs/2.build/3.near-components/environment.md [hunk 2]
```diff
@@ -3,6 +3,14 @@ id: dev-environment
 title: Choose your Dev Environment
 ---
 
+:::warning What is the state of BOS (NEAR Components)?
+
+We no longer recommend building on BOS due to its limited capabilities and discontinued security maintenance. Developers with active projects on BOS are encouraged to migrate to another deployment strategy.
+
+See [here](/build/web3-apps/frontend#bos-socialvm) for more info
```
Comment (bucanero) [inline docs/2.build/3.near-components/environment.md:10]:
```suggestion
See [here](/build/web3-apps/frontend#bos-socialvm) for more information.
```

File: docs/2.build/3.near-components/bos-gateway.md [hunk 3]
```diff
@@ -3,6 +3,14 @@ id: bos-gateway
 title: Using components on WebApps
 ---
 
+:::warning What is the state of BOS (NEAR Components)?
+
+We no longer recommend building on BOS due to its limited capabilities and discontinued security maintenance. Developers with active projects on BOS are encouraged to migrate to another deployment strategy.
+
+See [here](/build/web3-apps/frontend#bos-socialvm) for more info
```
Comment (bucanero) [inline docs/2.build/3.near-components/bos-gateway.md:10]:
```suggestion
See [here](/build/web3-apps/frontend#bos-socialvm) for more information.
```

File: docs/2.build/3.near-components/anatomy/web-methods.md [hunk 4]
```diff
@@ -5,6 +5,14 @@ title: Web Browser Methods
 
 import {WidgetEditor} from "@site/src/components/widget-editor"
 
+:::warning What is the state of BOS (NEAR Components)?
+
+We no longer recommend building on BOS due to its limited capabilities and discontinued security maintenance. Developers with active projects on BOS are encouraged to migrate to another deployment strategy.
+
+See [here](/build/web3-apps/frontend#bos-socialvm) for more info
```
Comment (bucanero) [inline docs/2.build/3.near-components/anatomy/web-methods.md:12]:
```suggestion
See [here](/build/web3-apps/frontend#bos-socialvm) for more information.
```

File: docs/2.build/3.near-components/anatomy/state.md [hunk 5]
```diff
@@ -6,6 +6,14 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import {WidgetEditor} from "@site/src/components/widget-editor"
 
+:::warning What is the state of BOS (NEAR Components)?
+
+We no longer recommend building on BOS due to its limited capabilities and discontinued security maintenance. Developers with active projects on BOS are encouraged to migrate to another deployment strategy.
+
+See [here](/build/web3-apps/frontend#bos-socialvm) for more info
```
Comment (bucanero) [inline docs/2.build/3.near-components/anatomy/state.md:13]:
```suggestion
See [here](/build/web3-apps/frontend#bos-socialvm) for more information.
```

File: docs/2.build/3.near-components/anatomy/social.md [hunk 6]
```diff
@@ -5,6 +5,14 @@ title: Social Interactions
 
 import {WidgetEditor} from "@site/src/components/widget-editor"
 
+:::warning What is the state of BOS (NEAR Components)?
+
+We no longer recommend building on BOS due to its limited capabilities and discontinued security maintenance. Developers with active projects on BOS are encouraged to migrate to another deployment strategy.
+
+See [here](/build/web3-apps/frontend#bos-socialvm) for more info
```
Comment (bucanero) [inline docs/2.build/3.near-components/anatomy/social.md:12]:
```suggestion
See [here](/build/web3-apps/frontend#bos-socialvm) for more information.
```

File: docs/2.build/3.near-components/anatomy/notifications.md [hunk 7]
```diff
@@ -6,6 +6,14 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import {WidgetEditor} from "@site/src/components/widget-editor"
 
+:::warning What is the state of BOS (NEAR Components)?
+
+We no longer recommend building on BOS due to its limited capabilities and discontinued security maintenance. Developers with active projects on BOS are encouraged to migrate to another deployment strategy.
+
+See [here](/build/web3-apps/frontend#bos-socialvm) for more info
```
Comment (bucanero) [inline docs/2.build/3.near-components/anatomy/notifications.md:13]:
```suggestion
See [here](/build/web3-apps/frontend#bos-socialvm) for more information.
```

File: docs/2.build/3.near-components/anatomy/near.md [hunk 8]
```diff
@@ -7,6 +7,14 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import {WidgetEditor} from "@site/src/components/widget-editor"
 
+:::warning What is the state of BOS (NEAR Components)?
+
+We no longer recommend building on BOS due to its limited capabilities and discontinued security maintenance. Developers with active projects on BOS are encouraged to migrate to another deployment strategy.
+
+See [here](/build/web3-apps/frontend#bos-socialvm) for more info
```
Comment (bucanero) [inline docs/2.build/3.near-components/anatomy/near.md:14]:
```suggestion
See [here](/build/web3-apps/frontend#bos-socialvm) for more information.
```

File: docs/2.build/3.near-components/anatomy/builtin-components.md [hunk 9]
```diff
@@ -3,6 +3,14 @@ id: builtin-components
 title: List of Native Components
 ---
 
+:::warning What is the state of BOS (NEAR Components)?
+
+We no longer recommend building on BOS due to its limited capabilities and discontinued security maintenance. Developers with active projects on BOS are encouraged to migrate to another deployment strategy.
+
+See [here](/build/web3-apps/frontend#bos-socialvm) for more info
```
Comment (bucanero) [inline docs/2.build/3.near-components/anatomy/builtin-components.md:10]:
```suggestion
See [here](/build/web3-apps/frontend#bos-socialvm) for more information.
```

File: docs/2.build/3.near-components/anatomy/bos-components.md [hunk 10]
```diff
@@ -4,6 +4,14 @@ title: Historical data
 sidebar_label: Handling Historical data
 ---
 
+:::warning What is the state of BOS (NEAR Components)?
+
+We no longer recommend building on BOS due to its limited capabilities and discontinued security maintenance. Developers with active projects on BOS are encouraged to migrate to another deployment strategy.
+
+See [here](/build/web3-apps/frontend#bos-socialvm) for more info
```
Comment (bucanero) [inline docs/2.build/3.near-components/anatomy/bos-components.md:11]:
```suggestion
See [here](/build/web3-apps/frontend#bos-socialvm) for more information.
```

File: docs/2.build/3.near-components/anatomy/bos-components.md [hunk 11]
```diff
@@ -4,6 +4,14 @@ title: Historical data
 sidebar_label: Handling Historical data
 ---
 
+:::warning What is the state of BOS (NEAR Components)?
+
+We no longer recommend building on BOS due to its limited capabilities and discontinued security maintenance. Developers with active projects on BOS are encouraged to migrate to another deployment strategy.
+
+See [here](/build/web3-apps/frontend#bos-socialvm) for more information.
```
Comment (bucanero) [inline docs/2.build/3.near-components/anatomy/bos-components.md:11]:
```suggestion
See the [Web3 frontends](/build/web3-apps/frontend#bos-socialvm) section for more information.
```

File: docs/2.build/3.near-components/anatomy/builtin-components.md [hunk 12]
```diff
@@ -3,6 +3,14 @@ id: builtin-components
 title: List of Native Components
 ---
 
+:::warning What is the state of BOS (NEAR Components)?
+
+We no longer recommend building on BOS due to its limited capabilities and discontinued security maintenance. Developers with active projects on BOS are encouraged to migrate to another deployment strategy.
+
+See [here](/build/web3-apps/frontend#bos-socialvm) for more information.
```
Comment (bucanero) [inline docs/2.build/3.near-components/anatomy/builtin-components.md:10]:
```suggestion
See the [Web3 frontends](/build/web3-apps/frontend#bos-socialvm) section for more information.
```

File: docs/2.build/3.near-components/anatomy/near.md [hunk 13]
```diff
@@ -7,6 +7,14 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import {WidgetEditor} from "@site/src/components/widget-editor"
 
+:::warning What is the state of BOS (NEAR Components)?
+
+We no longer recommend building on BOS due to its limited capabilities and discontinued security maintenance. Developers with active projects on BOS are encouraged to migrate to another deployment strategy.
+
+See [here](/build/web3-apps/frontend#bos-socialvm) for more information.
```
Comment (bucanero) [inline docs/2.build/3.near-components/anatomy/near.md:14]:
```suggestion
See the [Web3 frontends](/build/web3-apps/frontend#bos-socialvm) section for more information.
```

File: docs/2.build/3.near-components/anatomy/notifications.md [hunk 14]
```diff
@@ -6,6 +6,14 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import {WidgetEditor} from "@site/src/components/widget-editor"
 
+:::warning What is the state of BOS (NEAR Components)?
+
+We no longer recommend building on BOS due to its limited capabilities and discontinued security maintenance. Developers with active projects on BOS are encouraged to migrate to another deployment strategy.
+
+See [here](/build/web3-apps/frontend#bos-socialvm) for more information.
```
Comment (bucanero) [inline docs/2.build/3.near-components/anatomy/notifications.md:13]:
```suggestion
See the [Web3 frontends](/build/web3-apps/frontend#bos-socialvm) section for more information.
```

File: docs/2.build/3.near-components/anatomy/social.md [hunk 15]
```diff
@@ -5,6 +5,14 @@ title: Social Interactions
 
 import {WidgetEditor} from "@site/src/components/widget-editor"
 
+:::warning What is the state of BOS (NEAR Components)?
+
+We no longer recommend building on BOS due to its limited capabilities and discontinued security maintenance. Developers with active projects on BOS are encouraged to migrate to another deployment strategy.
+
+See [here](/build/web3-apps/frontend#bos-socialvm) for more information.
```
Comment (bucanero) [inline docs/2.build/3.near-components/anatomy/social.md:12]:
```suggestion
See the [Web3 frontends](/build/web3-apps/frontend#bos-socialvm) section for more information.
```

File: docs/2.build/3.near-components/anatomy/state.md [hunk 16]
```diff
@@ -6,6 +6,14 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import {WidgetEditor} from "@site/src/components/widget-editor"
 
+:::warning What is the state of BOS (NEAR Components)?
+
+We no longer recommend building on BOS due to its limited capabilities and discontinued security maintenance. Developers with active projects on BOS are encouraged to migrate to another deployment strategy.
+
+See [here](/build/web3-apps/frontend#bos-socialvm) for more information.
```
Comment (bucanero) [inline docs/2.build/3.near-components/anatomy/state.md:13]:
```suggestion
See the [Web3 frontends](/build/web3-apps/frontend#bos-socialvm) section for more information.
```

File: docs/2.build/3.near-components/anatomy/web-methods.md [hunk 17]
```diff
@@ -5,6 +5,14 @@ title: Web Browser Methods
 
 import {WidgetEditor} from "@site/src/components/widget-editor"
 
+:::warning What is the state of BOS (NEAR Components)?
+
+We no longer recommend building on BOS due to its limited capabilities and discontinued security maintenance. Developers with active projects on BOS are encouraged to migrate to another deployment strategy.
+
+See [here](/build/web3-apps/frontend#bos-socialvm) for more information.
```
Comment (bucanero) [inline docs/2.build/3.near-components/anatomy/web-methods.md:12]:
```suggestion
See the [Web3 frontends](/build/web3-apps/frontend#bos-socialvm) section for more information.
```

File: docs/2.build/3.near-components/bos-gateway.md [hunk 18]
```diff
@@ -3,6 +3,14 @@ id: bos-gateway
 title: Using components on WebApps
 ---
 
+:::warning What is the state of BOS (NEAR Components)?
+
+We no longer recommend building on BOS due to its limited capabilities and discontinued security maintenance. Developers with active projects on BOS are encouraged to migrate to another deployment strategy.
+
+See [here](/build/web3-apps/frontend#bos-socialvm) for more information.
```
Comment (bucanero) [inline docs/2.build/3.near-components/bos-gateway.md:10]:
```suggestion
See the [Web3 frontends](/build/web3-apps/frontend#bos-socialvm) section for more information.
```

File: docs/2.build/3.near-components/environment.md [hunk 19]
```diff
@@ -3,6 +3,14 @@ id: dev-environment
 title: Choose your Dev Environment
 ---
 
+:::warning What is the state of BOS (NEAR Components)?
+
+We no longer recommend building on BOS due to its limited capabilities and discontinued security maintenance. Developers with active projects on BOS are encouraged to migrate to another deployment strategy.
+
+See [here](/build/web3-apps/frontend#bos-socialvm) for more information.
```
Comment (bucanero) [inline docs/2.build/3.near-components/environment.md:10]:
```suggestion
See the [Web3 frontends](/build/web3-apps/frontend#bos-socialvm) section for more information.
```

File: docs/2.build/3.near-components/what-is.md [hunk 20]
```diff
@@ -8,6 +8,14 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import {WidgetEditor} from "@site/src/components/widget-editor"
 
+:::warning What is the state of BOS (NEAR Components)?
+
+We no longer recommend building on BOS due to its limited capabilities and discontinued security maintenance. Developers with active projects on BOS are encouraged to migrate to another deployment strategy.
+
+See [here](/build/web3-apps/frontend#bos-socialvm) for more information.
```
Comment (bucanero) [inline docs/2.build/3.near-components/what-is.md:15]:
```suggestion
See the [Web3 frontends](/build/web3-apps/frontend#bos-socialvm) section for more information.
```

**Comment by bucanero** [comment]
Hi @mpeterdev 👋 I've merged your changes and adjusted some minor details.

Let me know if you have more suggestions 🙏 thank you!

**Review by mpeterdev — APPROVED** [review]

**Pull request merged**